src/fonts/loadStatic.ts

```diff
@@ -0,0 +1,25 @@
+import Bravura from '../fonts/bravura';
+import Gonville from '../fonts/gonville';
+import Petaluma from '../fonts/petaluma';
+import Custom from '../fonts/custom';
```

Comment:
What's `custom`?

Reply:
I do not know. But it seems to be incomplete. Should I remove it?

Reply:
I am reviewing some recent font PRs while finishing up my own font PR. I encountered this question and also didn't know the answer. Here's my research.....

**Custom** is a set of 12 custom glyphs with outlines defined in `custom_glyphs.ts`:

```
'noteheadSquareWhite': {...},
'noteheadSquareBlack': {...},
'accidentalBuyukMucennebFlat': {...},
'accidentalBakiyeFlat': {...},
'accidentalKucukMucennebSharp': {...},
'accidentalKoron': {...},
'accidentalSori': {...},
'vexAccidentalMicrotonal2': {...},
'vexAccidentalMicrotonal3': {...},
'vexAccidentalMicrotonal4': {...},
'vexNoteHeadRectBlack': {...},
'vexNoteHeadRectWhite': {...}
```

It's possible there are some projects out there that use these glyphs, since multiple contributors have modified the custom font:
```
0xfe committed on Apr 7, 2020
https://github.com/0xfe/vexflow/commit/63ecb866d72401e495ff20f0bfb65c4cb9239157#diff-ed13115738ef399b956bc72c8845480f8ef376f7b3836fb66ae340e0a3122339

MarianneAK committed on Oct 20, 2019 
https://github.com/0xfe/vexflow/commit/160c5f56ff2dbf28e8a5e01cf51a1ec300233ed4#diff-ed13115738ef399b956bc72c8845480f8ef376f7b3836fb66ae340e0a3122339


PieterHartzer committed on Aug 29, 2018 
https://github.com/0xfe/vexflow/commit/c0ded252d2aab812eb5343b6be3b85551d2e0933
```

Are there SMuFL equivalents for these glyphs (e.g. in the Bravura font file)? If we can find suitable Bravura glyphs for each of these 12 glyphs, perhaps we can empty out the Custom font? (We could still keep support for it in case a developer wants to invent a glyph for his/her own purposes.)